### PR TITLE
Lockable fungible token example

### PR DESCRIPTION
**Pull request by itegulov**

### CODE REVIEW

**Comment by itegulov** [comment]
@ailisp Added some tests, this is ready for review

File: examples/lockable-fungible-token/__tests__/test-lockable-fungible-token.ava.js [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    // Prepare sandbox for tests, create accounts, deploy contracts, etx.
+    const root = worker.rootAccount;
+
+    // Deploy the jsvm contract.
+    const jsvm = await root.createAndDeploy(
+        root.getSubAccount('jsvm').accountId,
+        './node_modules/near-sdk-js/res/jsvm.wasm',
+    );
+
+    // Deploy status-message JS contract
```
Comment (ailisp) [inline examples/lockable-fungible-token/__tests__/test-lockable-fungible-token.ava.js:22]:
```suggestion
    // Deploy lockable FT JS contract
```

File: examples/lockable-fungible-token/__tests__/test-lockable-fungible-token.ava.js [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+    const lockableFt = await root.createSubAccount('status-message');
```
Comment (ailisp) [inline examples/lockable-fungible-token/__tests__/test-lockable-fungible-token.ava.js:23]:
```suggestion
    const lockableFt = await root.createSubAccount('lockable-ft');
```

**Review by ailisp — APPROVED** [review]
Looks good. It's a quite complicated contract with a number of tests!

**Pull request merged**